src/app/containers/BulletedList/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from 'react';
+import BulletedList from '@bbc/psammead-bulleted-list';
+import { ServiceContext } from '#contexts/ServiceContext';
+import Blocks from '../Blocks';
+import listItem from '../BulletedListItem';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+import { listPropTypes } from '#models/propTypes/list';
+
+const componentsToRender = { listItem };
+
+const BulletedListContainer = ({ blocks }) => {
+  const { script, service, dir } = useContext(ServiceContext);
+
+  return (
+    <GridItemConstrainedMedium>
+      <BulletedList script={script} service={service} dir={dir}>
+        <Blocks blocks={blocks} componentsToRender={componentsToRender} />
```

Comment:
Do we need to use the `Blocks` container here - is it doing anything helpful?

ie, could this be simplified to just rendering the `listItem` components directly?

Reply:
It could be simplified but we will have to repeat the validation and conversion of `{type, id, model: {blocks: []}}` into just `{blocks[]}`. I don't think we need to repeat the validation so I think I we need it.